euth/dashboard/views.py

```diff
@@ -120,10 +120,16 @@ class DashboardProjectListView(DashboardBaseMixin,
     filter_set = filters.ArchivedFilter
 
     def get_queryset(self):
-        return super().get_queryset().filter(
+        qs = super().get_queryset().filter(
             organisation=self.organisation
         )
 
+        for project in qs:
```

Comment:
Very good point. I fixed it by using `@property`. See the [corresponding a4 pull request](https://github.com/liqd/adhocracy4/pull/93) and https://github.com/liqd/a4-opin/commit/92f3401ec1e2e03db37cc299c735fc2bf5db50b8.